Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -28,4 +28,18 @@ - (id)initWithContact:(WACContact*)contact {
     return self;
 }
 
++ (NSDate*)convertDate:(WACContactDate*)date {
```

Comment:
+1, that would also better enforce private access to the helper.

---
In reply to: [75166689](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75166689) [](ancestors = 75166689,75164209,75146840)